src/Microsoft.Framework.Configuration.File/Helper/ConfigurationHelper.cs

```diff
@@ -10,18 +10,13 @@ public static class ConfigurationHelper
     {
         public static string ResolveConfigurationFilePath(IConfigurationBuilder configuration, string path)
```

Comment:
I think we should move this to FileConfigurationBuilderExtensions and make it an extension method, e.g. `ResolveFilePath(this IConfigurationBuilder configurationBuilder, string filePath)`.